tests/codegen/call_kstack.cpp

```diff
@@ -36,7 +36,7 @@ TEST(codegen, call_kstack_mapids)
   ast::CodegenLLVM codegen(driver.ctx, *bpftrace);
   bpftrace->bytecode_ = codegen.compile();
 
-  ASSERT_EQ(bpftrace->bytecode_.maps().size(), 8);
+  ASSERT_EQ(bpftrace->bytecode_.maps().size(), 7);
```

Comment:
I'm not sure I understand the point of this assert in the test. What if the spill rules change, and we need to update things. Why is `7` the right number here? I'm not really advocating the removal at this time, but I just want to gently introduce the idea of "`[change-detector tests](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html)`", which I think can add friction without a ton of value.

Reply:
> I'm not sure I understand the point of this assert in the test.

Me neither. I'm fine to remove it. Also if the right maps aren't being created we'll hear about it in a runtime or codegen test.